save-frontend/src/main/resources/locales/en/proposing.json

```diff
@@ -14,7 +14,7 @@
   "Affected projects": "Affected projects",
   "Propose a vulnerability": "Propose a vulnerability",
   "Project type": "Project type",
-  "Versions": "Versions",
+  "Affected versions": "Affected versions",
```

Comment:
The key should be renamed in the Chinese version, too.

Reply:
I deleted it so I wouldn't forget to add the correct translation later